public/main.js

```diff
@@ -81,28 +82,42 @@ function reloadTodoList() {
         todoListPlaceholder.style.display = "none";
         todos.forEach(function(todo) {
             var listItem = document.createElement("li");
-            var compButton = document.createElement("button");
-            compButton.onclick = function(event) { completeTodo(todo.id, reloadTodoList); };
-            compButton.innerHTML = "Complete";
-            compButton.classList.add("completeButton");
-            compButton.setAttribute("data-id", todo.id.toString());
-            var delButton = document.createElement("button");
-            delButton.onclick = function(event) { deleteTodo(todo.id, reloadTodoList); };
-            delButton.innerHTML = "Delete";
-            delButton.classList.add("deleteButton");
-            delButton.setAttribute("data-id", todo.id.toString());
+            var completeButton = document.createElement("button");
```

Comment:
This issue has actually been covered in a different feature branch, in which said function has already been created. fb0c2f916ad8a4a59100055b390272881a22d2df